stable/anchore-engine/README.md

```diff
@@ -244,6 +244,32 @@ See the anchore-engine [CHANGELOG](https://github.com/anchore/anchore-engine/blo
 A Helm post-upgrade hook job will shut down all previously running Anchore services and perform the Anchore database upgrade process using a Kubernetes job. 
 The upgrade will only be considered successful when this job completes successfully. Performing an upgrade will cause the Helm client to block until the upgrade job completes and the new Anchore service pods are started. To view progress of the upgrade process, tail the logs of the upgrade jobs `anchore-engine-upgrade` and `anchore-enterprise-upgrade`. These job resources will be removed upon a successful Helm upgrade.
 
+## Chart version 1.17.0
+
+Chart version 1.17.0 is a tightly scoped release, specifically aimed at enabling the Grype DB Builder within the Anchore Enterprise Feeds service. This upgrade will allow users to easily switch from the legacy vulnerability provider to the next-gen Grype vulnerability provider, without any vulnerability feed downtime. The impacts of this upgrade are as follows:
+
+* For deployments currently utilizing the legacy vulnerability provider, configured with `.Values.anchorePolicyEngine.vulnerabilityProvider=legacy`, this upgrade will enable the GrypeDB Builder feeds source on the Enterprise Feeds service.
```

Comment:
I'd prefer "GrypeDB driver" instead of "GrypeDB Builder" since that is more descriptive

Reply:
I like the use of GrypeDB driver, instead of Builder, but I don't recommend making a change in 1.17.  We need to make a change in Enterprise docs etc.   Let's plan to change terms in later helm versions?

Reply:
I'm only commenting on this specific description text for the README. Agree the chart content itself is a different discussion and should not be changed for 1.17